app/_src/policies/meshmetric.md

```diff
@@ -275,6 +283,62 @@ When you configure application scraping make sure to specify `application.name`
     This support will be added in the next release.
 - [Application](/docs/{{ page.version }}/policies/meshmetric/#applications) must expose metrics in Prometheus format for this integration to work
 
+{% endif_version %}
+
+{% if_version gte:2.7.x %}
+```yaml
+backends:
+  - type: OpenTelemetry
+    openTelemetry: 
+      endpoint: http://otel-collector.observability.svc:4317
+      refreshInterval: 60s
+```
+
+This configuration tells {{site.mesh_product_name}} data plane proxy to push metrics to [OpenTelemetry collector](https://opentelemetry.io/docs/collector/).
```

Comment:
What's the correct capitalisation / spelling of dataplane proxy? We have https://github.com/search?q=repo%3Akumahq%2Fkuma-website%20%22dataplane%20proxy%22&type=code and https://github.com/search?q=repo%3Akumahq%2Fkuma-website%20%22dataplane%20proxy%22&type=code

The previous paragraph uses this:

```suggestion
This configuration tells {{site.mesh_product_name}} Dataplane Proxy to push metrics to [OpenTelemetry collector](https://opentelemetry.io/docs/collector/).
```